app/styles/_notifications.less

```diff
@@ -0,0 +1,67 @@
+// fixes positioning that doesn't quite work w/core patternfly styles
+.navbar-pf-alt .nav .nav-item-iconic .badge {
```

Comment:
This rule will be obsoleted by changes in #1932.  @rhamilto, remove this after merge.